twitter_api/main.py

```diff
@@ -1,23 +1,315 @@
+# -*- coding: utf-8 -*-
+import json
 import sqlite3
 
-from flask import Flask
-from flask import g
+from datetime import datetime
+import dateutil.parser as du
+from flask import abort, Flask, g, request, Response
+from hashlib import md5
+from random import choice
+from string import ascii_lowercase, digits
+
+from .utils import auth_only, JSON_MIME_TYPE, json_only
+
 
 app = Flask(__name__)
 
 
+#############################
+# Helper functions:         #
+#############################
```

Comment:
I really liked how you guys created helper functions. Views are much cleaner when these are used.